public/class-ninja-forms-spn-addon-public.php

```diff
@@ -73,7 +73,7 @@ public function enqueue_styles() {
 		 * class.
 		 */
 
-		wp_enqueue_style( $this->plugin_name, plugin_dir_url( dirname( __FILE__ ) ) . 'vendor/intl-tel-input-master/build/css/intlTelInput.min.css', array(), $this->version, 'all' );
+		wp_enqueue_style( $this->plugin_name, plugin_dir_url( dirname( __FILE__ ) ) . 'node_modules/intl-tel-input/build/css/intlTelInput.min.css', array(), $this->version, 'all' );
```

Comment:
We don't need to enqueue this style via `wp_enqueue_style`.
This module should be imported via `@import` SASS feature